elyra/pipeline/processor_kfp.py

```diff
@@ -68,10 +69,15 @@ def process(self, pipeline):
 
             self.log.debug("Creating temp directory %s", temp_dir)
 
+            engine = runtime_configuration.metadata.get('engine')
+            print(f'>>> {engine}')
             # Compile the new pipeline
             try:
                 pipeline_function = lambda: self._cc_pipeline(pipeline, pipeline_name)  # nopep8 E731
-                kfp.compiler.Compiler().compile(pipeline_function, pipeline_path)
+                if ('argo' == engine or 'Argo' == engine):
```

Comment:
JSON enum values are case-sensitive, so there should never be a **validated** instance of runtime metadata that contains the value `"argo"`.  Are you seeing otherwise?

Reply:
No :)